### PR TITLE
Add docusaurus tool not pipeline logic

### PR DESCRIPTION
**Pull request by supertopher**
## What
Adds docusaurus tooling.  
**This will not actually change gitbook hosting**
This change adds the tooling to make it possible to use Github's branching magic via GH actions to publish to GitHub pages

Adding this now will help prevent docs merge conflicts going forward while tooling is worked out for publication

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,9 @@
...
 
-![](docs/.gitbook/assets/airbyte_new_logo.svg)
-
-**Data integration made simple, secure and extensible.**  
+**Data integration made simple, secure and extensible.**
 The new open-source standard to sync data from applications, APIs & databases to warehouses, lakes & other destinations.
 
-[![](docs/.gitbook/assets/deploy_locally.svg)](docs/deploying-airbyte/local-deployment.md) [![](docs/.gitbook/assets/deploy_on_aws.svg)](docs/deploying-airbyte/on-aws-ec2.md) [![](docs/.gitbook/assets/deploy_on_gcp.svg)](docs/deploying-airbyte/on-gcp-compute-engine.md) [![](docs/.gitbook/assets/deploy_on_k8s.svg)](docs/deploying-airbyte/on-kubernetes.md)
```
Comment (davinchia) [inline README.md:10]: why are these removed?

File: airbyte-config/init/src/main/resources/seed/destination_specs.yaml [hunk 2]
```diff
@@ -293,10 +293,10 @@
           type: "object"
           title: "Loading Method"
           description: "Loading method used to send select the way data will be uploaded\
-            \ to BigQuery. <br><b>Standard Inserts</b> - Direct uploading using SQL\
+            \ to BigQuery. <br/><b>Standard Inserts</b> - Direct uploading using SQL\
```
Comment (davinchia) [inline airbyte-config/init/src/main/resources/seed/destination_specs.yaml:296]: curious, docusarus doesn't like this formatting?

File: docusaurus/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Documentation and Docusaurus
+
+We use [docusaurus](https://docusaurus.io) for consitent process, in `Airbyte-Cloud` **no website is generated**.
```
Comment (davinchia) [inline docusaurus/README.md:3]:
```suggestion
We use [docusaurus](https://docusaurus.io) for consitent process, in `Airbyte` **no website is generated**.
```

File: docusaurus/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+cd docusaurus
+yarn install
+yarn build
+```
+
+At this point you will see any broken links that docusaurus was able to find.
+
+# Pull requests
+
+This is all just markdown at the end of the day so open a PR against master like any other commit
```
Comment (davinchia) [inline docusaurus/README.md:25]:
```suggestion
This is all just markdown at the end of the day so open a PR against master like any other commit.
```

File: docusaurus/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+# Local website
+If you want to run the docs as a docusaurus site locally just run `yarn run build`
```
Comment (davinchia) [inline docusaurus/README.md:28]:
```suggestion
If you want to run the docs as a docusaurus site locally just run `yarn run build`.
```

File: docusaurus/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+Url will show on successful command execution
```
Comment (davinchia) [inline docusaurus/README.md:29]:
```suggestion
Url will show on successful command execution.
```

File: docusaurus/docusaurus.config.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    tagline:
+        'Airbyte is an open-source data integration platform to build ELT pipelines. Consolidate your data in your data warehouses, lakes and databases.',
+    // this is non-functional, just a link back to OSS docs right now
+    url: 'https://docs.airbyte.com/',
+    baseUrl: '/',
+    onBrokenLinks: 'warn',
+    onBrokenMarkdownLinks: 'warn',
+    favicon: 'img/favicon.png',
+    organizationName: 'airbytehq', // Usually your GitHub org/user name.
+    projectName: 'airbyte-cloud', // Usually your repo name.
```
Comment (davinchia) [inline docusaurus/docusaurus.config.js:19]:
```suggestion
    projectName: 'airbyte', // Usually your repo name.
```

**Review by davinchia — APPROVED** [review]
:shipit: 

Some non-blocking comments.

File: airbyte-config/init/src/main/resources/seed/destination_specs.yaml (hunk 2 above)
Comment (supertopher) [reply airbyte-config/init/src/main/resources/seed/destination_specs.yaml:296]: JSX expects a closed tag and docusaurus uses JSX building static markdown.  So yes
